.github/workflows/test-coverage.yml

```diff
@@ -93,15 +92,18 @@ jobs:
           done
           echo "$(go env GOPATH)/bin" >> "$GITHUB_PATH"
           go install github.com/wadey/gocovmerge@latest
-          # shellcheck disable=SC2086
-          covs=$(find $modules -maxdepth 1 -type f \
-            -name coverage.out -print)
-          if [ -z "$covs" ]; then
+          
+          mapfile -t covs < <(
+            for m in $modules; do
```

Comment:
In the previous line, I believe that including `"` around `$modules` would already solve the SC2086. If so, I think the simpler command is better.